src/compat/e-132/SymArrayDmapCompat.chpl

```diff
@@ -40,7 +40,7 @@ module SymArrayDmapCompat
                 }
                 // fix the annoyance about boundingBox being empty
                 else {
-                  return {0..#0} dmapped blockDist(boundingBox={0..0});
+                  return dom dmapped blockDist(boundingBox=dom.expand(1));
```

Comment:
What is the point of this change? I thought there was a reason why it had to be the old way, but I guess I could have just made that up since I don't know what the reason would be.

Reply:
It fixes a compilation error I was seeing in multi-locale builds where the compiler couldn't figure out the return type of this function (`makeDistDom`). The first branch of this conditional calls `return blockDist.createDomain(dom)`, which can return a mutlidimensional domain, but this branch was always returning a single dimensional domain.

The new code still creates an empty domain with a non-empty bounding box but its rank matches the domain from the first branch of the conditional.